workspaces/mta/plugins/mta-frontend/package.json

```diff
@@ -57,16 +56,14 @@
     "@backstage/core-app-api": "^1.12.3",
     "@backstage/dev-utils": "^1.0.30",
     "@backstage/test-utils": "^1.5.3",
-    "@janus-idp/cli": "^1.11.1",
     "@testing-library/jest-dom": "^5.10.1",
     "@testing-library/react": "^12.1.3",
     "@testing-library/user-event": "^14.0.0",
     "msw": "^1.0.0"
   },
   "files": [
+    "app-config.dynamic.yaml",
```

Comment:
It contains just dynamic plugin content.

That's the reason why there is no "static" app-config[.example].yaml at the moment.